src/components/DatePickerContainer/DatePickerContainer.css

```diff
@@ -0,0 +1,11 @@
+.infoPanel {
+	padding: 10px 21px;
+	border-top: 1px solid #dce0e0;
+	color: #484848;
+}
+
+.info {
```

Comment:
The span for the exclamation point in the info panel. 
It's probably not necessary I just wanted the exclamation point to stand out a bit more. 
I can take it out if you want.